Initial Draft of Negotiation Mastery Program document in YAML file format

Computer8543 opened the pull request:
We can work together to fine-tune it.

billwestfall commented:
how about keeping one non implemented domain as an example of how a new one is created?

michael-basil commented:
That was the original thought. 

Perhaps Agile InnerSource is such a concept.

Maybe there is a different example. 

To me this is the utility of contending with the Void thought experiment.

Create a potential void. 

See what surfaces.

I also thought that our presentation covers the idea of other domains.

When someone comes in fresh do we want to influence and bias towards thinking of what else can be there or to engage what is already there first? 

The framing of the question I just posed suggests my bias of course.

billwestfall commented:
the void might be useful, I understand now  :)  👍

michael-basil commented:
I've moved this into review.   After conversations today with ISC I sense that the void will be more useful to allow that which should materialize come through organically and not overly prescriptive based on what we have done before.

I also planted the seed with @jimmanzzullo.

So I think if there is an approval this is an appropriate change to make understanding we are open to what may come back in following on whether it be Agile, InnerSource, Agile InnerSource or something we have not envisioned concretely in this conversation.

So @billwestfall (@jimmanzzullo @aphor @anuolu) or anyone else who has write access (in the Senpai team), with the approval I would merge it!

billwestfall reviewed (approved):
+1

Computer8543 committed:
Initial draft of Negotiation Mastery Program in a YAML file format.

Computer8543 committed:
Merge pull request #1 from Computer8543/kyle

Merge kyle branch into main branch

Computer8543 committed:
Merge Initial Draft code from main branch to begin negotiation. Also removed comments to make change so I can git commit directly to this issue.

Computer8543 committed:
Merge pull request #2 from Computer8543/main

Merge main branch changes into kyle branch

Computer8543 committed:
Merge branch 'mindset-dojo:main' into kyle

Computer8543 committed:
Create LICENSE.md

Add CC BY-SA 4.0 license to repo.

Computer8543 committed:
Create TODO.md

Create an initial draft of a minimal set of contribution guidelines for the Mindset Dojo repo.

Computer8543 committed:
Create README.md

Created README.md file to summarize the offerings of Mindset Dojo and it's community succinctly.

Computer8543 committed:
Merge pull request #3 from Computer8543/kyle

Merge kyle branch into main branch

Computer8543 committed:
Rename TODO.md to CONTRIBUTING.md

This is to follow open source naming conventions.

Computer8543 committed:
Merge pull request #4 from Computer8543/kyle

Rename TODO.md to CONTRIBUTING.md

Computer8543 committed:
Green Belt Push Request and Tag Change

Computer8543 committed:
Merge pull request #5 from Computer8543/kyle

Merge kyle branch into main
